resources/src/main/java/org/robolectric/res/android/CppAssetManager2.java

```diff
@@ -819,8 +818,8 @@ public boolean GetResourceName(int resid, ResourceName out_name) {
 //  boolean GetResourceFlags(int resid, int* out_flags);
   boolean GetResourceFlags(int resid, Ref<Integer> out_flags) {
     final Ref<FindEntryResult> entry = new Ref<>(null);
-    ApkAssetsCookie cookie = FindEntry(resid, (short) 0 /* density_override */,
-        false /* stop_at_first_match */, entry);
+    ApkAssetsCookie cookie =
+        FindEntry(resid, (short) 0 /* density_override */, /* stop_at_first_match */ entry);
```

Comment:
The `stop_at_first_match` comment is for `false` instead of `entry`.

Reply:
ohh yeah, thanks for correcting that. My dumb mistakes